### DISCUSSION TITLE
Only load 'Queries' from a generated Resolver (read-only resolver)

### DISCUSSION THREAD

**Discussion by rthijssen** [MichalLytek/typegraphql-prisma#447] [Q&A] [answered]
Hey all, I'd like to make an object read-only accessible via my API, does anyone know if it is possible to only load the queries from a generated CRUD resolver and thus not exposing any of the mutations?

**Comment by MichalLytek** [comment, marked as answer]
https://prisma.typegraphql.com/docs/advanced/exposing-actions

**Reply by rthijssen** [reply]
Guess I didn't have my coffee this morning and missed that in the docs. Thanks @MichalLytek .
